package.json

```diff
@@ -54,7 +54,7 @@
     "moment": "^2.22.1",
     "node-fetch": "^2.6.0",
     "openfin-service-signal": "^1.0.0",
-    "openfin-service-tooling": "1.0.10",
+    "openfin-service-tooling": "1.0.12-alpha.50",
```

Comment:
Did it in the small chance a change would be required to the tooling, so we wouldn't have to redeploy production.  The PR comment says it will change before PR is merged :).  Either way, we had to deploy to production anyways so its now gone.

Reply:
My bad, forgot to read the description. All good.